homeassistant/components/lupusec/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "lupusec",
   "name": "Lupus Electronics LUPUSEC",
   "documentation": "https://www.home-assistant.io/integrations/lupusec",
-  "requirements": ["lupupy==0.2.5"],
+  "requirements": ["lupupy==0.2.7"],
```

Comment:
Please make a github release corresponding to the PyPI release to make it easier for the community to track changes.

Reply:
Dony, please restart the pipeline :)

Reply:
Note that the version in the library is still 0.2.6:

https://github.com/majuss/lupupy/blob/d1e57cb74644a8685d22d4f74a64dc82583b0011/lupupy/constants.py#L3